### PR TITLE
TS EAs #4 batch 1 of 3

### PR DESCRIPTION
**Pull request by justinkaseman**
### Description
We missed a few External Adapters in the TS migration...

Breaking into smaller PRs for easier review

- [x] alphachain
- [x] bitex
- [x] bitso
- [x] coinlore

This one also adds some best practices to the `example` External Adapter.

### CODE REVIEW

File: bravenewcoin-vwap/package.json [hunk 1]
```diff
@@ -1,15 +0,0 @@
-{
```
Comment (emmick4) [inline bravenewcoin-vwap/package.json:1]: Lol how'd this get left behind?

File: coinlore/README.md [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
-Use the `API_DEFAULT_ENDPOINT` env variable to set the default API endpoint to use (either `dominance` or `globalmarketcap`). Defaults to `dominance`.
+---
+
+## Global Endpoint
 
 ### Input Params
 
-- `market`, `to`, `quote`: The coin to query (required)
-- `endpoint`: The endpoint to use (defaults to "dominance", one of "dominance")
+| Required? |  Name   |                                   Description                                   |   Options    | Defaults to |
```
Comment (emmick4) [inline coinlore/README.md:18]: Like these README updates, very clean.

File: coinlore/src/adapter.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  switch (endpoint.toLowerCase()) {
+    case 'dominance':
+    case global.NAME: {
+      return await global.execute(request, config)
+    }
+    case 'globalmarketcap': {
+      request.data.field = 'total_mcap'
+      return await global.execute(request, config)
+    }
```
Comment (emmick4) [inline coinlore/src/adapter.ts:27]: Looks like there could be something wrong here, does the 'dominance' case do anything?

**Review by emmick4 — COMMENTED** [review]
Just a question on the coinlore switch, other than that looks great. Almost like you've done these once or twice 😉

File: bravenewcoin-vwap/package.json (hunk 1 above)
Comment (justinkaseman) [reply bravenewcoin-vwap/package.json:1]: :man_shrugging:

**Review by emmick4 — APPROVED** [review]

**Pull request merged**